quickstart/deploy-mayastor.md

```diff
@@ -6,317 +6,102 @@
-{% endtabs %}
-
-Verify that the CSI-controller pod is running.
+Run the following command to discover all the _stable versions_ of the added chart repository:
 
 {% tabs %}
 {% tab title="Command" %}
 ```text
-kubectl get pods -n mayastor --selector=app=csi-controller
+helm search repo mayastor --versions
 ```
 {% endtab %}
-
-{% tab title="Example Output" %}
+{% tab title="Sample Output" %}
 ```text
-NAME                             READY   STATUS    RESTARTS   AGE
-csi-controller-579f77f64-7dq7g   3/3     Running   0          39m
+ NAME             	CHART VERSION	APP VERSION  	DESCRIPTION                       
+mayastor/mayastor	v2.0.0        	v2.0.0       	Mayastor Helm chart for Kubernetes
 ```
 {% endtab %}
 {% endtabs %}
 
-### MSP Operator
- 
-{% tabs %}
-{% tab title="Command \(GitHub Latest\)" %}
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/v1.0.2/deploy/msp-deployment.yaml
-```
-{% endtab %}
-{% endtabs %}
+{% hint style="info" %}
+To discover all the versions (including unstable versions), execute:
+`helm search repo mayastor --devel --versions`
+{% endhint %}
 
-Verify that the pool operator pod is running.
 
+3. Run the following command to install Mayastor _version 2.0_.
 {% tabs %}
 {% tab title="Command" %}
 ```text
-kubectl get pods -n mayastor --selector=app=msp-operator
-```
-{% endtab %}
-
-{% tab title="Example Output" %}
-```text
-NAME                            READY   STATUS    RESTARTS   AGE
-msp-operator-7849d59fcd-mcw5b   1/1     Running   0          21s
-```
-{% endtab %}
-{% endtabs %}
-
-
-### Data Plane
-
-{% tabs %}
-{% tab title="Command \(GitHub Latest\)" %}
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/master/v1.0.2/mayastor-daemonset.yaml
+helm install mayastor mayastor/mayastor -n mayastor --create-namespace
 ```
 {% endtab %}
-{% endtabs %}
-
-Verify that the Mayastor DaemonSet has been correctly deployed. The reported Desired, Ready and Available instance counts should be equal, and should match the count of worker nodes which carry the label `openebs.io/engine=mayastor` \(as performed earlier in the "[Preparing the Cluster](preparing-the-cluster.md#label-the-storage-nodes)" stage\).
-
-{% tabs %}
-{% tab title="Command" %}
+{% tab title="Sample Output" %}
 ```text
-kubectl -n mayastor get daemonset mayastor
-```
-{% endtab %}
+NAME: mayastor
+LAST DEPLOYED: Thu Sep 22 18:59:56 2022
+NAMESPACE: mayastor
+STATUS: deployed
+REVISION: 1
+NOTES:
+OpenEBS Mayastor has been installed. Check its status by running:
+$ kubectl get pods -n mayastor
 
-{% tab title="Example Output" %}
-```text
-NAME       DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                                         AGE
-mayastor   3         3         3       3            3           kubernetes.io/arch=amd64,openebs.io/engine=mayastor   108s
+For more information or to view the documentation, visit our website at https://openebs.io.
```

Comment:
As you've suggested I'm changing this link to https://mayastor.gitbook.io/introduction/